state/peerAccountsDB.go

```diff
@@ -62,10 +63,18 @@ func (adb *PeerAccountsDB) SnapshotState(rootHash []byte) {
 	log.Trace("peerAccountsDB.SnapshotState", "root hash", rootHash)
 	trieStorageManager := adb.mainTrie.GetStorageManager()
 
+	stats := &snapshotStatistics{
+		wg:        &sync.WaitGroup{},
+		startTime: time.Now(),
+	}
+
 	trieStorageManager.EnterPruningBufferingMode()
-	trieStorageManager.TakeSnapshot(rootHash, true, nil)
+	stats.NewSnapshotStarted()
+	trieStorageManager.TakeSnapshot(rootHash, nil, stats)
 	trieStorageManager.ExitPruningBufferingMode()
 
+	go printStats(stats, "snapshotState peer trie", rootHash)
+
```

Comment:
let's have it here the blocking from import-db also

Reply:
Done.